actor/state/state_async_provider.go

```diff
@@ -75,6 +75,11 @@ func (d *DaprStateAsyncProvider) Apply(actorType, actorID string, changes []*Act
 			Value:         value,
 		})
 	}
+
+	if len(operations) == 0 {
```

Comment:
Please add this case in tests.

Reply:
@daixiang0 It seems the tests are not running, In my local development environment with `go test -v ./actor/state` for example covers my changes:

![image](https://user-images.githubusercontent.com/1398228/146135642-f174f98f-88ea-489a-bcc8-7956e6fc3549.png)

Reply:
This test can be flaky :(

Reply:
> This test can be flaky :(

![image](https://user-images.githubusercontent.com/1398228/147734244-b03c145b-f65b-462f-b2d1-74cc225e2a5f.png)

Personally, I strongly doubt it's a codecov issue which reports wrong line coverage (it should covers `if len(operations) == 0`  but wrongly calculated to wrong lines, which will never be executed)

I suggest to store the coverage output file as artifact, the compare to the codecov reports.

Reply:
I agree but I am not familiar with codecov, @artursouza do you know who can make it better?